docs/usage.md

```diff
@@ -42,7 +42,11 @@ The following query parameters can be used when `reqMode` is set to `http` or is
 |srcPort      |The source (entry) port of a service. Useful only when specifying multiple destinations of a single service. The parameter can be prefixed with an index thus allowing definition of multiple destinations for a single service (e.g. `srcPort.1`, `srcPort.2`, and so on).|No| |80|
 |templateBePath|The path to the template representing a snippet of the backend configuration. If specified, the backend template will be loaded from the specified file. If specified, `templateFePath` must be set as well. See the [Templates](#templates) section for more info.| | |/tmpl/be.tmpl|
 |templateFePath|The path to the template representing a snippet of the frontend configuration. If specified, the frontend template will be loaded from the specified file. If specified, `templateBePath` must be set as well. See the [Templates](#templates) section for more info.| | |/tmpl/fe.tmpl|
-|users        |A comma-separated list of credentials(<user>:<pass>) for HTTP basic auth, which applies only to the service that will be reconfigured.|No| |usr1:pwd1,usr2:pwd2|
+|users        |A comma-separated list of credentials(<user>:<pass>) for HTTP basic auth, which applies only to the service that will be reconfigured. If used with usersSecret or when USERS environment variable is set password may be omitted - it will be taken from usersSecret file or global configuration if usersSecret is not present. |No| |usr1:pwd1,usr2:pwd2|
+|usersSecret  |Name of file prefixed /run/secrets/dfp_users_%s from which credentials will be taken for this service. /run/secrets/dfp_users_* files must be a comma-separated list of credentials(<user>:<pass>) |No| |monitoring|
```

Comment:
Do we need this variable. If the prefix is `dfp_users_`, we can find the files ourselves. It could be something similar to https://github.com/vfarcic/docker-flow-proxy/blob/master/proxy/ha_proxy.go#L62. It's a bit slower to go through files but it is easier from user's perspective. Just add a secret and nothing else.